### ISSUE TITLE
Getting Started

### ISSUE THREAD

**Issue by bscully** [mdgoldberg/sportsref#1]
I understand the project is largely a WIP but any scripts to start downloading boxscore and play-by-play data?

**Comment by mdgoldberg** [comment]
@bscully yep, that is possible to do at the moment. I can show you how - any sport in particular you're interested in?

**Comment by bscully** [comment]
I’m interested in the NFL.  I modified some code and got it working.  Got a few data errors here and there.
With a list of BoxScoreIDs (as variable boxes) I ran this code:

    all_gms, broken = pd.DataFrame(), []
    for gm in boxes:
        print gm
        b = BoxScore(gm)
        try:
            df = pbp.expand_details(b.pbp())
            all_gms = all_gms.append( df )
        except:
            broken.append(gm)
        del b

It was pretty memory intensive, 3 years would take up ~10 GB so I broke-up the load.
I haven’t analyzed the output yet but having this much play by play data is amazing.

Very nice package!
